packages/sdks/javascript/Makefile

```
@@ -0,0 +1,7 @@
+test:
+	@docker build --force-rm -f Dockerfile-dev -t devopness_sdk-javascript .
+	@docker run devopness_sdk-javascript npm run test
+
+build:
```

Target comments are useful so they can be listed by the `help` target:
```suggestion
build: ## Build Docker image
```

---

@jfoliveira resolved